widget/playground/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -38,6 +38,11 @@ export function MultiSelect(props: MuliSelectPropTypes) {
   const hasValue = !valueAll;
   const showMore = hasValue && value.length > MAX_CHIPS;
   const onBack = () => setShowNextModal(false);
+  const onOpenNextModal = () => {
+    if (!props.disabled) {
```

Comment:
Should you kindly also do this for the single select (default blockchain).

Reply:
done